Readme update (#28)
TikoyX/devUps: README.md

# MiniTwit

> MiniTwit is a dashboard/blogging platoform where users can make posts. Originally made in Python, it is currently in 

Reviewer: typo

> MiniTwit is a dashboard/blogging platoform where users can make posts. Originally made in Python, it is currently in 

Reviewer 2: ```suggestion
MiniTwit is a dashboard/blogging platform where users can make posts. Originally made in Python with Flask, it is currently in 
```

the process of being reworked to a C#/React web-application. This project is part of the course DevOps at IT-Univeristy of Copenhagen, 2023. 

## Installation on Windows/Linux/macOS

> Assuming you have Git Bash and Docker. Run the follwing in Git Bash or other preferred shell. 

Reviewer 2: ```suggestion
You need Docker installed for this project. To install this look at [Docker](https://www.docker.com).
After ensuring Docker is installed clone the repository:
```

> ```Git Bash

Reviewer 2: ```suggestion
```shell
```

git clone https://github.com/TikoyX/devUps.git
```
From the directory, then run the follwing command with Docker running. 

> ```Git Bash

Reviewer 2: ```suggestion
```shell
```

docker-compose up
```
A port for frontend and backend should appear. Try open port 3000 on local to see
the application.  

(Should this give problems with prior docker-files try to run "docker-compose build --no-cache")

## Usage

> To have an active development environment, we suggest running the frontend directoiry with "npm start" and downloading 

Reviewer 2: ```suggestion
To have an active development environment, run the following command inside the *minitwit-frontend* directory `npm start`.
```

> the proper extensions for any development. 

Reviewer 2: ```suggestion

```

## Contributing

> For contribution guidelines, see contributing.md. 

Reviewer 2: ```suggestion
For contribution guidelines, see [contributing.md](contributing.md). 
```
